src/nupic/engine/__init__.py

```diff
@@ -49,7 +49,7 @@
     ("nupic.regions.KNNClassifierRegion", "KNNClassifierRegion"),
     ("nupic.regions.PluggableEncoderSensor", "PluggableEncoderSensor"),
     ("nupic.regions.PyRegion", "PyRegion"),
-    ("nupic.regions.RecordSensor", "RecordSensor"),
```

Comment:
@rhyolight: @scottpurdy is right. See my PR that demonstrated the same change that you're attempting to make and built successfully in all environments: https://github.com/numenta/nupic/pull/3632/files.

Please compare your changes against my PR or reopen my PR and use it, instead.